Configure.cmake

```diff
@@ -600,6 +636,46 @@ if (ENFORCE_VSX AND NOT COMPILER_SUPPORTS_VSX)
   message(FATAL_ERROR "ENFORCE_VSX is specified and that feature is disabled or not supported by the compiler")
 endif()
 
+# IBM Z
+
+option(DISABLE_Z13 "Disable Z13" OFF)
+option(ENFORCE_Z13 "Build fails if Z13 is not supported by the compiler" OFF)
+
+if(SLEEF_ARCH_S390X AND NOT DISABLE_Z13)
+  set (CMAKE_REQUIRED_FLAGS ${FLAGS_ENABLE_Z13})
+  CHECK_C_SOURCE_COMPILES("
+  #include <vecintrin.h>
+  #define VECTOR __attribute__((vector_size(16)))
+  int main() {
+    VECTOR double d;
+    d = __builtin_s390_vfidb(d, 4, 4);
+  }"
+    COMPILER_SUPPORTS_Z13)
+endif()
+
+if (ENFORCE_Z13 AND NOT COMPILER_SUPPORTS_Z13)
+  message(FATAL_ERROR "ENFORCE_Z13 is specified and that feature is disabled or not supported by the compiler")
+endif()
+
+option(DISABLE_Z14 "Disable Z14" OFF)
+option(ENFORCE_Z14 "Build fails if Z14 is not supported by the compiler" OFF)
+
+if(SLEEF_ARCH_S390X AND NOT DISABLE_Z14)
+  set (CMAKE_REQUIRED_FLAGS ${FLAGS_ENABLE_Z14})
+  CHECK_C_SOURCE_COMPILES("
+  #include <vecintrin.h>
+  #define VECTOR __attribute__((vector_size(16)))
+  int main() {
+    VECTOR double d;
+    d = __builtin_s390_vec_sqrt(d);
```

Comment:
Z13 and Z14 both have sqrt support for double-precision, I guess you should check single-precision instead.

Reply:
Done.